sphinx/util/logging.py

```diff
@@ -65,6 +65,12 @@ def getLogger(name):
     Sphinx logger always uses ``sphinx.*`` namesapce to be independent from
     settings of root logger.  It enables to log stably even if 3rd party
     extension or imported application resets logger settings.
+
+    Example usage:
+
+    >>> from sphinx.utils import logging
+    >>> logger = logging.getLogger(__name__)
+    >>> logger.info('Hello, this is an extension!')
```

Comment:
Why console style? I prefer to use code-block.

Reply:
How about following instead?
```
Example usage::

    from sphinx.utils import logging
    ...
```

Reply:
I put it in a code block, but I kept the console style because it's the only way to show cause-effect (i.e. if you run these commands, you will get this output)